workorders.lic

```diff
@@ -76,9 +71,29 @@ class WorkOrders
         shape_items(info, item, quantity)
       end
     when 'carving'
+      @material_type = @use_bone ? 'bone' : 'stone'
       @engineering_room = settings.engineering_room
       tools = settings.carving_tools
       @belt = settings.engineering_belt
+      
+      if !@use_bone 
```

Comment:
I mean that the code in the `if` block seems identical to the code in the `else` block.